testsuite/MDAnalysisTests/test_imports.py

```diff
@@ -34,6 +40,11 @@ def _run_test_relative_import(testing_module):
                         "module {testing_module} at linenumber {lineno}.".format(**vars()))
 
     def test_relative_imports(self):
-        list_testing_modules = glob.glob(os.path.join(self.path_to_testing_modules, '*.py'))
-        for testing_module in list_testing_modules:
-            yield self._run_test_relative_import, testing_module
+        for dirpath, dirnames, files in os.walk(self.path_to_testing_modules):
```

Comment:
This should probably get span out into a new PR if this is fixing something? @mnmelo